data/json/furniture.json

```diff
@@ -313,6 +328,7 @@
         "id" : "f_oven",
         "name": "oven",
         "symbol": "#",
+        "description": "Used for heating and cooking food. Doesn't look like it's working..",
```

Comment:
Should probably mention it's electric, not gas. Possibly suggest dismantling for parts.